pkgs/by-name/te/television/wrapper.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildEnv,
+  television,
+  targetPackages,
+  makeBinaryWrapper,
+}:
+
+lib.makeOverridable (
+  pkgs: f:
+  buildEnv {
+    inherit (lib.appendToName "with-pkgs" television) name;
+    inherit (television) meta;
+
+    paths = f pkgs;
```

Ideally we'd use this in a `lib.makeBinaryPath` call and set `$PATH` that way. It'd avoid most of the oddities below

We should also probably have television itself linked here

```suggestion
    paths = [ television ];
```